src/runtime/manifest-parser.peg

```diff
@@ -864,23 +861,34 @@ RecipeParticle
     });
   }
 
-RecipeParticleItem = RecipeParticleSlotConnection / RecipeParticleConnection
-
 // TODO(cypher1): s/dir/direction/ to match slot connections and the rest of the code base
 RecipeParticleConnection
-  = param:NameWithColon? dir:Direction target:(whiteSpace ParticleConnectionTargetComponents)? eolWhiteSpace dependentConnections:(Indent (SameIndent RecipeParticleConnection)*)?
+  = param:NameWithColon? direction:((SlotDirection / Direction) whiteSpace?) target:ParticleConnectionTargetComponents? eolWhiteSpace dependentConnections:(Indent (SameIndent RecipeParticleConnection)*)?
   {
+    direction = optional(direction, d => d[0], null);
```

Comment:
Looks like the `optional()` isn't needed - `direction` isn't being optionally matched in the parse expression?